### ISSUE TITLE
Support Rakkas (Vite/React framework) - Dev server

### ISSUE THREAD

**Issue by birkskyum** [denoland/deno#20852]
Rakkas - [Website](https://rakkasjs.org/) - [GitHub](https://github.com/rakkasjs/rakkasjs)

**Platform**

Darwin 23.0.0 arm64 arm

**Version**

deno 1.37.1

**Repro**

- `deno run -A  npm:create-rakkas-app@latest my-rakkas-app`
- `proceed with default selection`
- `cd my-rakkas-app`
- `DENO_FUTURE=1 deno task dev`

**Expected**

Launch of demo app

```sh
   RAKKAS  0.6.20 development server is running 💃 (ready in 442 ms)

  ➜  Local:   http://localhost:5173/
  ➜  Network: use --host to expose
```

**Actual**

When I go to the http://localhost:5173/, this is what I see

```sh
➜ deno task dev
Task dev rakkas

   RAKKAS  0.6.20 development server is running 💃 (ready in 830 ms)

  ➜  Local:   http://localhost:5173/
  ➜  Network: use --host to expose
7:00:37 PM [vite] Error when evaluating SSR module /node_modules/.deno/rakkasjs@0.6.20/node_modules/rakkasjs/dist/server.js:
|- TypeError: Cannot read properties of undefined (reading 'env')
    at ../../node_modules/.pnpm/prop-types@15.8.1/node_modules/prop-types/index.js (/Users/admin/repos/deno-kitchensink/my-rakkas-app/node_modules/.deno/rakkasjs@0.6.20/node_modules/rakkasjs/dist/server.js, <anonymous>:994:17)
    at __require (/Users/admin/repos/deno-kitchensink/my-rakkas-app/node_modules/.deno/rakkasjs@0.6.20/node_modules/rakkasjs/dist/server.js, <anonymous>:41:50)
    at eval (/Users/admin/repos/deno-kitchensink/my-rakkas-app/node_modules/.deno/rakkasjs@0.6.20/node_modules/rakkasjs/dist/server.js, <anonymous>:1395:33)
    at async instantiateModule (file:///Users/admin/repos/deno-kitchensink/my-rakkas-app/node_modules/.deno/rakkasjs@0.6.20/node_modules/vite/dist/node/chunks/dep-52909643.js:56052:9)

```

**Additional information**

- https://github.com/denoland/deno/issues/24061

The production build works:

- `deno task build`
- `deno task start` (even todo example works)

**Comment by nathanwhit** [comment]
It looks like this was fixed with #25291, it now runs without issue for me on 2.0.0-rc.2.

**Comment by birkskyum** [comment]
Oh, intersting. 

When I go to the Todo page, and interacting with it, it'll prompt network errors in the console:

<img width="699" alt="Screenshot 2024-09-18 at 01 31 16" src="https://github.com/user-attachments/assets/dc29c525-db9a-4a06-b83f-7ed7971b98fd">
